src/middleware/validateSchema.js

```diff
@@ -23,44 +20,36 @@ module.exports = (shouldUseJoiError = false) => {
 	// return the validation middleware
 	// eslint-disable-next-line consistent-return
 	return (req, res, next) => {
-		const route = req.route.path
-		const method = req.method.toLowerCase()
-
-		if (includes(supportedMethods, method) && has(schemas, route)) {
-			// get schema for the current route
-			const schema = get(schemas, route)
 
-			if (schema) {
-				// Validate req.body using the schema and validation options
-				const { value, error } = schema
-					.validate(req.body, validationOptions)
-
-				if (error) {
-					// Joi Error
-					const JoiError = {
-						status: 'failed',
-						error: {
-							original: error._object,
-							// fetch only message and type from each error
-							message: map(error.details, ({ message }) => (
-								message.replace(/['"]/g, '')
-							)).join(', ')
-						}
+		if (schema) {
+			// Validate req.body using the schema and validation options
+			const { value, error } = schema
+				.validate(req.body, validationOptions)
```

Comment:
we want to validate the `req` as a whole not just `req.body`. this will help with validating the params as well like validating the id in `/articles/:id`